e2e/test/scenarios/actions/model-actions.cy.spec.js

```diff
@@ -328,6 +333,51 @@ describe(
       cy.findByLabelText("101").should("not.exist");
       cy.findByLabelText("ID").should("be.visible");
     });
+
+    it("should show detailed form errors for constraint violations when executing model actions", () => {
+      const actionName = "Update";
+
+      cy.get("@modelId").then(modelId => {
+        cy.visit(`/model/${modelId}/detail`);
+        cy.wait("@getModel");
+      });
+
+      cy.findByRole("tablist").within(() => {
+        cy.findByText("Actions").click();
+      });
+
+      cy.wait([
+        "@fetchMetadata",
+        "@fetchMetadata",
+        "@fetchMetadata",
+        "@getModelAction",
+        "@getSearchResults",
+      ]);
+
+      createBasicActions();
```

Comment:
Do you think using `createImplicitActions` which is used in `e2e/test/scenarios/actions/actions-in-object-detail-view.cy.spec.js` would be a better alternative since that one uses API and doesn't rely on the UI?

You should be able to use `createImplicitActions({ modelId });` inside `cy.get("@modelId").then` before visiting the model detail page.

I'm not sure about all the wait from line `349-354`, but maybe we can also remove some of that as well (I guess some of that are required because we use `createBasicActions()` which relies on the UI

Reply:
280c65ca4ca2f38d431c8880139308b31212401d